Automattic/wp-calypso
# A4A: Add welcome message above signup form for users coming from the client plugin

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/48

## Proposed Changes

* When the `source=client-plugin` query string is present on the `/signup` page, renders a "You haven't signed up yet!" banner. 
* This occurs when a user installs the A4A client plugin, connects their site, and attempts to visit the dashboard via clicking a link in the plugin.

## Testing Instructions



* Grab a JN site with Jetpack Beta installed.
* Activate the Automattic for Agencies Client plugin in Jetpack Beta.
* Using an account that is not a part of the agencies program, connect your site using the A4A client plugin.
* Click the "Visit the Dashboard" button in the A4A plugin.
* Replace the base URL of the button with your A4A live link or other test URL.
* Verify you are redirected to the /signup page.
* Verify the banner is displayed.
* Test the banner on different screen sizes.

## Pre-merge Checklist




## Screenshots

<img width="1064" alt="Screenshot 2024-04-18 at 6 39 36 PM" src="https://github.com/Automattic/wp-calypso/assets/10933065/0db186e8-53ea-4d28-a188-a2cd07baa163">
